### ISSUE TITLE
Unable to install to PCL targeting .NETStandard1.4

### ISSUE THREAD

**Issue by DerPeit** [Fody/PropertyChanged#185]
I recently updated my PCL libraries to the .NETStandard1.4 target. Unfortunately I am no longer able to install the NuGet package now due to incompatibilities:

```
PropertyChanged.Fody 1.51.3 is not compatible with .NETStandard,Version=v1.4.
Some packages are not compatible with .NETStandard,Version=v1.4.
```

Is it possible to update the `nuspec` to support the netstandard targets as well?

**Comment by SebastianStehle** [comment]
I have the same problem, tested several versions of .netstandard

**Comment by ViktorBergman** [comment]
This worked for me:
[Installing a PCL into netstandard Libraries ](http://motzcod.es/post/148657853472/installing-a-pcl-into-netstandard-libraries)

**Comment by ravensorb** [comment]
Any chance of just updated the nuspec to also indicate which netstandard profiles it supports as well?

**Comment by xalikoutis** [comment]
If you reference "Microsoft.NETCore.Portable.Compatibility": "1.0.1" PropertyChanged.Fody will not complain any more. But if you reference your .net standard(pcl) project to an Android for Xamarin in order to implement Xamarin.Form, Android wont compile with the following

> provides a compile-time reference assembly for mscorlib on MonoAndroid,Version=v6.0, but there is no run-time assembly compatible with win.
> 
> error CS1703: Multiple assemblies with equivalent identity have been imported

**Comment by DerPeit** [comment]
At least for me none of the hints worked. I tried back and forth several times without success. I am removing this package now entirely.

**Comment by irsali** [comment]
Using below in project.json worked for me
"frameworks": {
"netstandard1.5": {
"imports": "dnxcore50"
}
}

**Comment by SimonCropp** [comment]
fixed in  2.0.0-beta001
